README.md

```diff
@@ -33,6 +33,10 @@ There could be two reasons for this problem
 
 We welcome you to get involved in development, documentation, testing the OpenSearch Dashboards reports plugin. See our [CONTRIBUTING.md](./CONTRIBUTING.md) and join in.
 
+## Notifications Integration
+
+OpenSearch Dashboards Reports integration with [Notifications](https://github.com/opensearch-project/notifications) is currently in progress, tracking for the 8/30 OpenSearch 1.1 release.
```

Comment:
Can you also review the first paragraph and change the wording if needed. I assume the overview paragraph should clearly says the feature we currently supported, what we don't support, and what we plan to support. Now I think they are mixing together

Reply:
Added